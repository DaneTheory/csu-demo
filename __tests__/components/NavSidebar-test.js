import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router, Route } from 'react-router-dom';

import NavSidebar from '../../src/js/components/NavSidebar';
import store from '../../src/js/store';

// needed because this:
// https://github.com/facebook/jest/issues/1353
jest.mock('react-dom');

test('NavSidebar renders', () => {
  const component = renderer.create(
    <Provider store={store}>
      <Router>
        <Route path='/' component={NavSidebar} />
      </Router>
    </Provider>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
