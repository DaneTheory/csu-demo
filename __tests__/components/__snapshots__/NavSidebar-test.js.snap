exports[`test NavSidebar renders 1`] = `null`;
